event/listener.php

```diff
@@ -115,7 +115,7 @@ public function show_page_links()
 
 			// Assign any available custom icons for the current link in the user's style
 			$custom_icon = '';
-			foreach ($pages_icons as $icon_path => $ext_name)
+			foreach (array_keys($pages_icons) as $icon_path)
```

Comment:
This is probably going to have a tiny perf hit though, I'm not sure it serves any purpose.